.travis.yml

```diff
@@ -1,16 +1,25 @@
 ---
+dist: trusty
+sudo: false
+
 cache:
   directories:
     - "$(npm config get prefix)/bin"
 
 language: python
 python: '2.7'
 
+addons:
+  apt:
```

Comment:
Huh. In my previous project, I only needed to set the distro to Trusty. I'll look into it.